Qs.Presentation.WebApi/Properties/launchSettings.json

```diff
@@ -0,0 +1,31 @@
+﻿{
+  "$schema": "https://json.schemastore.org/launchsettings.json",
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+      "applicationUrl": "http://localhost:3595",
+      "sslPort": 44387
+    }
+  },
+  "profiles": {
+    "qs_web_api": {
```

Comment:
Update this name to a more user friendly one: example `QS: Web API`. This is what a developer sees in their IDE when they want to run the app.

Reply:
Changed to “QS: Web API”